README.md

```diff
@@ -75,15 +75,24 @@ The above `nginx` configuration proxies `https://localhost:4437` to `http://loca
 
 ### Start emulator
 
-```
+```bash
 npm run dev
 ```
 
 ### Shared storage
 
-To run this demo, [follow the instructions](https://developer.chrome.com/docs/privacy-sandbox/shared-storage/#try-the-shared-storage-api) for enabling the Privacy Sandbox Ads APIs experiment flag at **chrome://flags/#privacy-sandbox-ads-apis**.
+To run this demo, enable the experiment flags:
+
+* chrome://flags/#enable-fenced-frames
+* chrome://flags/#privacy-sandbox-enrollment-overrides
+
+Then, open Google Chrome from the command line with the additional arguments as below to disable the need for attestations when running locally.
```

Comment:
Then, open [Google Chrome from the command line](https://www.chromium.org/developers/how-tos/run-chromium-with-flags/) with the...